v_3_3_2/master_template.go

```diff
@@ -0,0 +1,2316 @@
+package v_3_3_1
```

Comment:
I don't know much about the process, but the package seems to be `v_3_3_1` everywhere in the `v_3_3_2` folder. Or is that intentional?

Reply:
Yes, the copy PR is just to create the new files. This makes the PR easier to review. Next PR will make v_3_3_2 the new WIP and I'll fix the imports then.

Like vendor PRs the reviews on copy PRs are for information. So you can merge without a review. The process is basically the same as for new version bundles.

Reply:
Ah alright. Thanks for the enlightment! :bulb: